docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.install

```diff
@@ -246,3 +246,37 @@ function hs_paragraph_types_update_10006() {
     }
   }
 }
+
+/**
+ * Moves drag & drop action for paragraphs from 3 dot menu to paragraph toolbar.
+ */
+function hs_paragraph_types_update_10007() {
+  $config_to_update = [
+    'core.entity_form_display.node.hs_basic_page.default',
+    'core.entity_form_display.node.hs_private_page.default',
+  ];
+  // Load the configuration from the config/sync directory.
+  $config_directory = new FileStorage(Settings::get('config_sync_directory'));
+
+  foreach ($config_to_update as $config) {
+    $config_data = $config_directory->read($config);
+
+    // Check if the form display configuration already exists.
+    $entity_type_manager = \Drupal::entityTypeManager();
+    /** @var ConfigEntityStorageInterface $config_storage */
+    $config_storage = $entity_type_manager->getStorage('entity_form_display');
+    $existing_config = $config_storage->load($config_data['id']);
+
+    if ($existing_config) {
+      foreach ($config_data as $key => $value) {
+        $existing_config->set($key, $value);
+      }
+      $existing_config->save();
```

Comment:
```suggestion
      $existing_config = $config_storage
        ->updateFromStorageRecord($existing_config, $config_data);
      $existing_config->save();
```

The [updateFromStorageRecord](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Config%21Entity%21ConfigEntityStorageInterface.php/function/ConfigEntityStorageInterface%3A%3AupdateFromStorageRecord/10) should work better here.

Reply:
Done!